docs/manual-deployment.md

```diff
@@ -33,43 +33,11 @@ cp devops/.env.sample devops/.env
 | `IMAGE_TAG` | The default tag for Docker images that will be pushed to the container registry and deployed with the Azure TRE. |
 | `ACR_NAME` | A globally unique name for the Azure Container Registry (ACR) that will be created to store deployment images. |
 | `ARM_SUBSCRIPTION_ID` | *Optional for manual deployment.* The Azure subscription ID for all resources. |
-| `RESOURCE_PROCESSOR_CLIENT_ID` | The client (app) ID of a service principal with "Owner" role to the subscription as created above. Used by Resource Processor Function to deploy workspaces and workspace services. |
-| `RESOURCE_PROCESSOR_CLIENT_SECRET` | The client secret (app password) of a service principal with "Onwer" role to the subscription as created above. Used by the depl09oyment processor function to deploy workspaces and workspace services. |
+| `ARM_TENANT_ID` | MISSING? |
```

Comment:
In the sample we have:
```
ARM_TENANT_ID=__CHANGE_ME__
ARM_CLIENT_ID=__CHANGE_ME__
ARM_CLIENT_SECRET=__CHANGE_ME__
```

They either all need to be here, or if using "logged in credentials", none?

Reply:
done